tests/integration/kubernetes/confidential_common.sh

```diff
@@ -7,6 +7,9 @@
 
 source "${BATS_TEST_DIRNAME}/tests_common.sh"
 
+SUPPORTED_TEE_HYPERVISORS=("qemu-sev" "qemu-snp" "qemu-tdx" "qemu-se")
+SUPPORTED_NON_TEE_HYPERVISORS=("qemu")
+
 function setup_unencrypted_confidential_pod() {
 	get_pod_config_dir
```

Comment:
for NON-TEE should we set image annotations?

Reply:
@GabyCT Yes, as @wainersm and @fitzthum said in your [PR](https://github.com/kata-containers/kata-containers/pull/9137), we need to set the image annotation to pull encrypted image using confidential rootfs in the future work.